test/e2e/airgap.go

```diff
@@ -22,3 +23,15 @@ func runVSphereAirgapConfigFlow(test *framework.ClusterE2ETest, localCIDRs strin
 	)
 	test.DeleteCluster()
 }
+
+func runDockerAirgapConfigFlow(test *framework.ClusterE2ETest) {
+	test.GenerateClusterConfig()
+	test.DownloadArtifacts()
+	test.ExtractDownloadedArtifacts()
+	test.DownloadImages()
+	test.ImportImages()
```

Comment:
Since we now separate the downloading and Importing images, I think we should run ImportImages() after airgapping